tests/ui/type-alias-impl-trait/normalize-alias-type.rs

```diff
@@ -0,0 +1,32 @@
+// check-pass
+#![feature(type_alias_impl_trait)]
+
+pub trait Tr {
+    fn get(&self) -> u32;
+}
+
+impl Tr for (((u32,),),) {
+    #[inline]
```

Comment:
Is this test complete? The original ICE only triggers with `-Zmir-opt-level=3` but I don't see this in the test :see_no_evil:

Reply:
ah good catch, thanks!